x-pack/legacy/plugins/apm/common/__snapshots__/elasticsearch_fieldnames.test.ts.snap

```diff
@@ -310,4 +314,6 @@ exports[`Transaction TRANSACTION_TYPE 1`] = `"transaction type"`;
 
 exports[`Transaction URL_FULL 1`] = `"http://www.elastic.co"`;
 
+exports[`Transaction USER_AGENT_NAME 1`] = `undefined`;
+
```

Comment:
This test is... not great. And it's totally on me (I introduced it!). 
LMK if you have any ideas to improve it.

Reply:
Actually, it would be useful if you added a user agent in the sample doc:

https://github.com/elastic/kibana/blob/51a6eeb6c2a2798a3724b412fe8e4e3c0ac4bbf2/x-pack/legacy/plugins/apm/common/elasticsearch_fieldnames.test.ts#L36

Reply:
Added user agent in e865ebe1b5.